torchgeo/models/weights.py

```diff
@@ -0,0 +1,154 @@
+"""Pretrained weights for models."""
```

Comment:
These should be in `torchgeo/models/resnet.py`, this is where they live in [torchvision](https://github.com/pytorch/vision/blob/main/torchvision/models/resnet.py)

Reply:
Okay, so if we have weights available for a new model, for example some transformer, we would create `torchgeo/models/some_transformer.py` and add the WeightsEnum there?

Reply:
on the note of resnet.py, do you still want to keep the torchgeo resnet implementation in there? It feels not that useful, since we have timm models for the different tasks and the pretrained weights we add now are also based on timm.

Reply:
Doesn't matter to me whether we use torchvision or timm, as long as a user can easily load a ResNet model with our pre-trained weights.